dev/all_tests.yaml

```diff
@@ -340,4 +340,4 @@ tests:
     api: gate
     args:
       stage_name: randomWait
-      stage_params: '{ "maxWaitTime": 5 }'
+      stage_params: "'{ \"maxWaitTime\": 5 }'"
```

Comment:
Thank you for sticking through this with me.

YAML `"'{ \"maxWaitTime\": 5 }'"` becomes Python `'\'{ "maxWaitTime": 5 }\''` and is written to the shell as `'{ "maxWaitTime": 5 }'`

Reply:
😭 😭 😭 😭 😭 😭 😭 😭 😭 😭 😭 
But this looks correct!